Zend/zend_language_scanner.l

```diff
@@ -517,6 +517,8 @@ ZEND_API int open_file_for_scanning(zend_file_handle *file_handle)
 	zend_string *compiled_filename;
 
 	if (zend_stream_fixup(file_handle, &buf, &size) == FAILURE) {
+		/* Still add it to open_files so the file_handle destruction logic works correctly. */
+		zend_llist_add_element(&CG(open_files), file_handle);
```

Comment:
I see that this was already merged as part of other change ( https://github.com/php/php-src/commit/2f648444952433f3f4907b4749cb713f6d033e66 ) so it should be removed from this PR.